src/vs/monaco.d.ts

```diff
@@ -3172,6 +3172,16 @@ declare namespace monaco.editor {
 		 * Defaults to false.
 		 */
 		originalEditable?: boolean;
+		/**
+		 * Original editor should be have code lens enabled?
+		 * Defaults to false.
+		 */
+		originalCodeLens?: boolean;
+		/**
+		 * Original editor should be have code lens enabled?
```

Comment:
typo: should be "Modified editor" here

Reply:
oops, copy/paste... :/ good catch, thanks!

Reply:
no problem